Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/calendar/CalendarCalculationTest.kt

```diff
@@ -0,0 +1,70 @@
+package de.rki.coronawarnapp.ui.calendar
+
+import junit.framework.TestCase
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
+import org.junit.Test
+
+class CalendarCalculationTest {
+
+    private var pattern = "dd.MM.yyyy"
+
+    @Test
+    fun calculateSameYearSameMonth() {
+        var input = "27.08.2020"
+        val dateTime =
+            DateTime.parse(input, DateTimeFormat.forPattern(pattern))
+        val dates = CalendarCalculation().getDates(dateTime)
+
+        // First day - 3 of August
+        TestCase.assertEquals(3, dates.first().date.dayOfMonth)
```

Comment:
Please use kotest for assertions, i.e. `dates.first().date.dayOfMonth shouldBe 3`

Reply:
why is that necessary?

Reply:
Right now that is our only test that uses TestCase. I would argue for kotest or JUnit Assertions to keep in line with our other test suite components